### PR TITLE
Dbyrdaquent seo working branch

### PR DESCRIPTION
**Pull request by dbyrdaquent**
Have changed 14 documents thus far, need someone to check and merge.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/configure-dynamic-emergency-calling.md [hunk 1]
```diff
@@ -91,14 +91,14 @@ For more information about configuring emergency addresses, see [Add an emergenc
 
 Network settings are used to determine the location of a Teams client, and to dynamically obtain emergency calling policies and an emergency location. You can configure network settings according to how your organization wants emergency calling to function.
 
-Network settings include sites that include a collection of subnets--these are used exclusively for dynamic policy assignment to users.  For example, a TeamsEmergencyCalling Policy and TeamsEmergencyCallRouting Policy might be assigned to the “Redmond site” so that any user that roams from home or another Microsoft location is configured with emergency numbers, routing, and security desk specific to Redmond.  
+Network settings include sites that include a collection of subnets--these are used exclusively for dynamic policy assignment to users.  For example, a TeamsEmergencyCalling Policy and TeamsEmergencyCallRouting Policy might be assigned to the "Redmond site" so that any user that roams from home or another Microsoft location is configured with emergency numbers, routing, and security desk specific to Redmond.  
 
 >[!Note]
```
Comment (JohanFreelancer9) [inline Teams/configure-dynamic-emergency-calling.md]:
```suggestion
> [!NOTE]
```

File: Teams/configure-dynamic-emergency-calling.md [hunk 2]
```diff
@@ -91,14 +91,14 @@ For more information about configuring emergency addresses, see [Add an emergenc
...
 >Subnets can also be defined in LIS and can be associated with an emergency location.
```
Comment (JohanFreelancer9) [inline Teams/configure-dynamic-emergency-calling.md]:
```suggestion
> Subnets can also be defined in LIS and can be associated with an emergency location.  
```

File: Teams/configure-dynamic-emergency-calling.md [hunk 3]
```diff
@@ -91,14 +91,14 @@ For more information about configuring emergency addresses, see [Add an emergenc
...
 
 Keep the following definitions in mind:
 
-- Trusted IP’s contain a collection of the Internet external IPs of the enterprise network and are used to determine if the user’s endpoint is inside the corporate network. An attempt to obtain a dynamic policy or location will only be made if the user’s external IP matches an IP in the Trusted IP address. A match can be made against either IPv4 or IPv6 IP addresses and is dependent upon the format of the IP packet sent to the network settings.  (If a public IP address has both IPv4 and IPv6, you need to add both as trusted IP addresses.)
+- Trusted IP's contain a collection of the Internet external IPs of the enterprise network and are used to determine if the user's endpoint is inside the corporate network. An attempt to obtain a dynamic policy or location will only be made if the user's external IP matches an IP in the Trusted IP address. A match can be made against either IPv4 or IPv6 IP addresses and is dependent upon the format of the IP packet sent to the network settings.  (If a public IP address has both IPv4 and IPv6, you need to add both as trusted IP addresses.)
```
Comment (JohanFreelancer9) [inline Teams/configure-dynamic-emergency-calling.md]:
```suggestion
- Trusted IP's contain a collection of the Internet external IPs of the enterprise network and are used to determine if the user's endpoint is inside the corporate network. An attempt to obtain a dynamic policy or location will only be made if the user's external IP matches an IP in the Trusted IP address. A match can be made against either IPv4 or IPv6 IP addresses and is dependent upon the format of the IP packet sent to the network settings. (If a public IP address has both IPv4 and IPv6, you need to add both as trusted IP addresses.)
```

File: Teams/expand-teams-across-your-org/shifts/plan-to-move-staffhub-teams-to-shifts-in-teams.md [hunk 4]
```diff
@@ -22,7 +22,7 @@ appliesto:
...
+> Effective December 31, 2019, Microsoft StaffHub will be retired. We're building StaffHub capabilities into Microsoft Teams. Today, Teams includes the Shifts app for schedule management and additional capabilities will roll out over time. StaffHub will stop working for all users on December 31, 2019. Anyone who tries to open StaffHub will be shown a message directing them to download Teams. To learn more, see [Microsoft StaffHub to be retired](microsoft-staffhub-to-be-retired.md). 
 
 Making the transition from StaffHub to Teams begins when you start planning for the change. To help ensure your move to Teams is successful, we've created a sample  timeline that demonstrates a typical transition plan. The sample timeline outlines planning activities to prepare for the move and takes you through to moving your organization's StaffHub teams to Teams.
```
Comment (JohanFreelancer9) [inline Teams/expand-teams-across-your-org/shifts/plan-to-move-staffhub-teams-to-shifts-in-teams.md]:
```suggestion
Making the transition from StaffHub to Teams begins when you start planning for the change. To help ensure your move to Teams is successful, we've created a sample timeline that demonstrates a typical transition plan. The sample timeline outlines planning activities to prepare for the move and takes you through to moving your organization's StaffHub teams to Teams.
```

File: Teams/migration-interop-guidance-for-teams-with-skype.md [hunk 5]
```diff
@@ -43,23 +43,23 @@ As an organization with Skype for Business starts to adopt Teams, administrators
...
     - If the user is homed on-premises, use `Move-CsUser` from the on-premises admin tools to first move the user to Skype for Business Online.  If you have Skype for Business Server 2019 or CU8 for Skype for Business Server 2015, you can specify the `-MoveToTeams` switch in `Move-CsUser` to move the user directly to Teams as part of the move online. This option will also migrate the user's meetings to Teams. If `-MoveToTeams` is not specified or not available, then after `Move-CsUser` completes, assign TeamsOnly mode to that user using either PowerShell or the Teams Admin Center. For more details see [Move users between on-premises and cloud](https://docs.microsoft.com/skypeforbusiness/hybrid/move-users-between-on-premises-and-cloud).  For more details on meeting migration, see [Using the Meeting Migration Service (MMS)](https://docs.microsoft.com/skypeforbusiness/audio-conferencing-in-office-365/setting-up-the-meeting-migration-service-mms).
```
Comment (JohanFreelancer9) [inline Teams/migration-interop-guidance-for-teams-with-skype.md]:
```suggestion
    - If the user is homed on-premises, use `Move-CsUser` from the on-premises admin tools to first move the user to Skype for Business Online.  If you have Skype for Business Server 2019 or CU8 for Skype for Business Server 2015, you can specify the `-MoveToTeams` switch in `Move-CsUser` to move the user directly to Teams as part of the move online. This option will also migrate the user's meetings to Teams. If `-MoveToTeams` is not specified or not available, then after `Move-CsUser` completes, assign TeamsOnly mode to that user using either PowerShell or the Teams Admin Center. For more details, see [Move users between on-premises and cloud](https://docs.microsoft.com/skypeforbusiness/hybrid/move-users-between-on-premises-and-cloud). For more details on meeting migration, see [Using the Meeting Migration Service (MMS)](https://docs.microsoft.com/skypeforbusiness/audio-conferencing-in-office-365/setting-up-the-meeting-migration-service-mms).
```

File: Teams/migration-interop-guidance-for-teams-with-skype.md [hunk 6]
```diff
@@ -148,7 +148,7 @@ TeamsUpgradePolicy governs routing for incoming federated chats and calls. Feder
...
+When a user is in any of the Skype for Business modes (SfBOnly, SfBWithTeamsCollab, SfBWithTeamsCollabAndMeetings), all incoming chats and calls are routed to the user's Skype for Business client. To avoid end user confusion and ensure proper routing, calling and chat functionality in the Teams client is automatically disabled when a user is in any of the Skype for Business modes. Similarly, meeting scheduling in Teams is automatically disabled when users are in the SfBOnly or SfBWithTeamsCollab modes, and automatically enabled when a user is in the SfBWithTeamsCollabAndMeetings mode. For details, see [Teams client experience and conformance to coexistence modes](https://docs.microsoft.com/MicrosoftTeams/teams-client-experience-and-conformance-to-coexistence-modes).
 
 > [!Note]
```
Comment (JohanFreelancer9) [inline Teams/migration-interop-guidance-for-teams-with-skype.md]:
```suggestion
> [!NOTE] 
```

File: Teams/migration-interop-guidance-for-teams-with-skype.md [hunk 7]
```diff
@@ -148,7 +148,7 @@ TeamsUpgradePolicy governs routing for incoming federated chats and calls. Feder
...
+When a user is in any of the Skype for Business modes (SfBOnly, SfBWithTeamsCollab, SfBWithTeamsCollabAndMeetings), all incoming chats and calls are routed to the user's Skype for Business client. To avoid end user confusion and ensure proper routing, calling and chat functionality in the Teams client is automatically disabled when a user is in any of the Skype for Business modes. Similarly, meeting scheduling in Teams is automatically disabled when users are in the SfBOnly or SfBWithTeamsCollab modes, and automatically enabled when a user is in the SfBWithTeamsCollabAndMeetings mode. For details, see [Teams client experience and conformance to coexistence modes](https://docs.microsoft.com/MicrosoftTeams/teams-client-experience-and-conformance-to-coexistence-modes).
 
 > [!Note] 
 > - Prior to delivery of the automatic enforcement of Teams and Channels, the SfbOnly and SfBWithTeamsCollab modes behave the same.
```
Comment (JohanFreelancer9) [inline Teams/migration-interop-guidance-for-teams-with-skype.md]:
```suggestion
> Prior to delivery of the automatic enforcement of Teams and Channels, the SfbOnly and SfBWithTeamsCollab modes behave the same.
```

File: Teams/migration-interop-guidance-for-teams-with-skype.md [hunk 8]
```diff
@@ -164,7 +164,7 @@ When a user is in any of the Skype for Business modes (SfBOnly, SfBWithTeamsColl
...
 |**SfBWithTeamsCollab**|A user runs both Skype for Business and Teams side-by-side. This user:</br><ul><li>Has the functionality of a user in SfBOnly mode.<li>Has Teams enabled only for group collaboration (Channels); chat/calling/meeting scheduling are disabled.</ul>|
 |**SfBWithTeamsCollab</br>AndMeetings**|A user runs both Skype for Business and Teams side-by-side. This user:<ul><li>Has the chat and calling functionality of user in SfBOnly mode.<li>Has Teams enabled for group collaboration (channels - includes channel conversations); chat and calling are disabled.<li>Can schedule only Teams meetings, but can join Skype for Business or Teams meetings.</ul>|
```
Comment (JohanFreelancer9) [inline Teams/migration-interop-guidance-for-teams-with-skype.md]:
```suggestion
|**SfBWithTeamsCollab</br>AndMeetings**|A user runs both Skype for Business and Teams side-by-side. This user:<ul><li>Has the chat and calling functionality of a user in SfBOnly mode.<li>Has Teams enabled for group collaboration (channels - includes channel conversations); chat and calling are disabled.<li>Can schedule only Teams meetings, but can join Skype for Business or Teams meetings.</ul>|
```

File: Teams/upgrade-basic.md [hunk 9]
```diff
@@ -73,7 +73,7 @@ Senior leaders are accountable for company success; be sure to keep them in the
 Teams offers compatible Skype for Business functionality—such as IM (chat) and meetings—but it can also do so much more. As a true hub for teamwork, Teams enables workgroups to manage projects, files, conversations, and apps all in one location. By default, Teams is turned on for all organizations. Decide how your organization will use Teams and configure your environment for success. 
 
 > [!Note]
```
Comment (JohanFreelancer9) [inline Teams/upgrade-basic.md]:
```suggestion
> [!NOTE]
```

**Comment by scanum** [comment]
@LolaJacobsen PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by DaniHalfin** [comment]
@dbyrdaquent - please make sure to open PRs into seo-working-branch.
also, can you merge from seo-working-branch into your fork and resolve conflicts?

thanks :)

**Comment by dbyrdaquent** [comment]
Hello,

The merge option does not appear in GitHub Desktop for me.

- David

**Pull request merged**